op-chain-ops/script/cheatcodes_utilities.go

```diff
@@ -233,6 +233,17 @@ func (c *CheatCodesPrecompile) ParseTomlAddress_65e7c844(tomlStr string, key str
 	panic("should never get here")
 }
 
+func (c *CheatCodesPrecompile) ComputeCreate2Address_890c283b(salt, codeHash [32]byte) (common.Address, error) {
```

Comment:
How did you know to generate the function name as `ComputeCreate2Address_890c283b`?

It surprised me that this is the first time we've needed the create2 cheatcode implemented.

Reply:
this lgtm, I compared with some test vectors outlined in the eip: https://eips.ethereum.org/EIPS/eip-1014 

I had to make the deployer address change:

```solidity
func (c *CheatCodesPrecompile) ComputeCreate2Address_890c283b(salt, codeHash [32]byte) (common.Address, error) {
	data := make([]byte, 1+20+32+32)
	data[0] = 0xff
	deployerAddress := common.HexToAddress("0x0000000000000000000000000000000000000000")
	copy(data[1:], deployerAddress.Bytes())
	copy(data[1+20:], salt[:])
	copy(data[1+20+32:], codeHash[:])
	finalHash := crypto.Keccak256(data)
	// Take the last 20 bytes of the hash to get the address
	return common.BytesToAddress(finalHash[12:]), nil
}
```

Reply:
> How did you know to generate the function name as ComputeCreate2Address_890c283b?

Go doesn't support method overloading, so to allow the runner to call the correct overloads we suffix them with the 4 byte selector. `890c283b` is the 4byte from the Forge library for `ComputeCreate2Address(bytes32, bytes32)`.

Reply:
This particular method always sets the sender to the deterministic deployer, which matches how Forge works.